src/components/PeopleManagement/AddMembersModalContent.jsx

```diff
@@ -0,0 +1,145 @@
+import { useEnterpriseLearners } from '../learner-credit-management/data';
+
+const AddMembersModalContent = ({
+  onEmailAddressesChange,
+  isGroupInvite,
+  enterpriseUUID,
+  groupName,
+  enterpriseGroupLearners,
+}) => {
+  const [learnerEmails, setLearnerEmails] = useState([]);
+  const [emailAddressesInputValue, setEmailAddressesInputValue] = useState('');
+  const [memberInviteMetadata, setMemberInviteMetadata] = useState({
+    isValidInput: null,
+    lowerCasedEmails: [],
+    duplicateEmails: [],
+    emailsNotInOrg: [],
+  });
+  const { allEnterpriseLearners } = useEnterpriseLearners({ enterpriseUUID });
+
+  const handleAddMembersBulkAction = useCallback((value) => {
+    if (!value) {
+      setLearnerEmails([]);
+      onEmailAddressesChange([]);
+      return;
+    }
+    setLearnerEmails(prev => [...prev, ...value]);
+  }, [onEmailAddressesChange]);
+
+  const handleRemoveMembersBulkAction = useCallback((value) => {
+    if (!value) {
+      setLearnerEmails([]);
+      onEmailAddressesChange([]);
+      return;
+    }
+    setLearnerEmails(prev => prev.filter((el) => !value.includes(el)));
+  }, [onEmailAddressesChange]);
+
+  const handleEmailAddressesChanged = useCallback((value) => {
+    if (!value) {
+      setLearnerEmails([]);
+      onEmailAddressesChange([]);
+      return;
+    }
+    // handles csv upload value and formats emails into an array of strings
+    const emails = value.split('\n').map((email) => email.trim()).filter((email) => email.length > 0);
+    setLearnerEmails(emails);
+  }, [onEmailAddressesChange]);
+
+  const debouncedHandleEmailAddressesChanged = useMemo(
+    () => debounce(handleEmailAddressesChanged, EMAIL_ADDRESSES_INPUT_VALUE_DEBOUNCE_DELAY),
+    [handleEmailAddressesChanged],
+  );
+
+  useEffect(() => {
+    debouncedHandleEmailAddressesChanged(emailAddressesInputValue);
+  }, [emailAddressesInputValue, debouncedHandleEmailAddressesChanged]);
+
+  // Validate the learner emails emails from user input whenever it changes
+  useEffect(() => {
+    const inviteMetadata = isInviteEmailAddressesInputValueValid({
+      learnerEmails,
+      allEnterpriseLearners,
+    });
+    setMemberInviteMetadata(inviteMetadata);
+    if (inviteMetadata.canInvite) {
+      onEmailAddressesChange(learnerEmails, { canInvite: true });
+    } else {
+      onEmailAddressesChange([]);
+    }
+  }, [onEmailAddressesChange, learnerEmails, allEnterpriseLearners]);
+
+  return (
+    <Container size="lg" className="py-3">
+      <h3>
+        <FormattedMessage
+          id="people.management.page.create.group.section.header"
+          defaultMessage="Create a custom group of members"
+          description="Header for the section to create a new group."
+        />
+      </h3>
+      <Row>
+        <Col>
+          <h4 className="mt-4">Add new members to your group</h4>
+          <p>Only members registered with your organization can be added to your group. Learn more</p>
```

Comment:
Even though we might not have the help center doc yet, I think we could make this a hyperlink that just points to edx.org or something